api/src/main/kotlin/edu/wgu/osmt/api/model/ApiSkillV2.kt

```diff
@@ -0,0 +1,120 @@
+package edu.wgu.osmt.api.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import edu.wgu.osmt.collection.Collection
+import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.db.JobCodeLevel
+import edu.wgu.osmt.db.PublishStatus
+import edu.wgu.osmt.richskill.RichSkillDescriptor
+import edu.wgu.osmt.richskill.RichSkillDescriptorDao
+import edu.wgu.osmt.util.OsmtUtil.Companion.parseMultiValueToSingleValue
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+@JsonInclude(JsonInclude.Include.ALWAYS)
+class ApiSkillV2(private val rsd: RichSkillDescriptor, private val cs: Set<Collection>, private val appConfig: AppConfig) {
+
+    @JsonProperty("@context")
+    val context = appConfig.rsdContextUrl
+
+
+    @JsonProperty
+    val `type` = "RichSkillDescriptor"
+
+    @get:JsonProperty
+    val creator: String
+        get() = appConfig.defaultCreatorUri
+
+    @get:JsonProperty
+    val author: String?
+        get() = parseMultiValueToSingleValue(rsd.authors.map { "${it.value}," }.toString())
+
+    @get:JsonProperty
+    val status: PublishStatus
+        get() = rsd.publishStatus()
+
+    @get:JsonProperty
+    val creationDate: ZonedDateTime
+        get() = rsd.creationDate.atZone(ZoneId.of("UTC"))
+
+    @get:JsonProperty
+    val updateDate: ZonedDateTime
+        get() = rsd.updateDate.atZone(ZoneId.of("UTC"))
+
+    @get:JsonProperty
+    val publishDate: ZonedDateTime?
+        get() = rsd.publishDate?.atZone(ZoneId.of("UTC"))
+
+    @get:JsonProperty
+    val archiveDate: ZonedDateTime?
+        get() = rsd.archiveDate?.atZone(ZoneId.of("UTC"))
+
+    @get:JsonProperty
+    val skillName: String
+        get() = rsd.name
+
+    @get:JsonProperty
+    val skillStatement: String
+        get() = rsd.statement
+
+    @get:JsonProperty
+    val keywords: List<String>
+        get() = rsd.searchingKeywords.mapNotNull { it.value }
+
+    @get:JsonProperty
+    val category: String?
+        get() = parseMultiValueToSingleValue(rsd.categories.map { "${it.value}," }.toString())
+
+    @get:JsonProperty
+    val id: String
+        get() = rsd.canonicalUrl(appConfig.baseUrl)
+
+    @get:JsonProperty
+    val uuid: String
+        get() = rsd.uuid.toString()
```

Comment:
Seems uuid is string, is to string redundant?

Reply:
it is indeed. but this was removed also from this implementation.